test/hmpps/components/court-cases-release-dates/offence-card/offenceCard.test.ts

```diff
@@ -115,7 +115,7 @@ describe('Tests for offence card component', () => {
         'Licence period': '5 years 6 months 0 weeks 0 days',
         'Custodial term': '1 years 0 months 0 weeks 0 days',
         'Sentence type': 'SDS (Standard Determinate Sentence)',
-        'Sentence date': '2024-09-22',
+        'Sentencing warrant date': '2024-09-22',
```

Comment:
This is actually just the warrant date (aka appearance date). There's a separate conversation ongoing as to why this is even in the offence card as it belongs to the appearance (a level up)

But for the time being the decision is to re-label to indicate it's the warrant date